tests/utils/shippable/sanity.sh

```diff
@@ -15,10 +15,9 @@ fi
 
 case "${group}" in
     1) options=(--skip-test pylint --skip-test ansible-doc --skip-test validate-modules) ;;
-    2) options=(                   --test      ansible-doc                             ) ;;
-    3) options=(                                                --test validate-modules) ;;
-    4) options=(--test pylint --exclude tests/unit/ --exclude plugins/module_utils/) ;;
-    5) options=(--test pylint           tests/unit/           plugins/module_utils/) ;;
+    2) options=(                   --test      ansible-doc      --test validate-modules) ;;
```

Comment:
Nice.
Since `validate-modules` only takes 1 minute we could maybe combine that with group3. Though lets revisit that in the future.

I've already got a note to review and rebalance the various test suites in Shippable.